recipes-kernel/linux/linux-at91-5.4.104/sam9x60/0002-Commented_serial_dbg_messages.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/drivers/staging/wilc1000/debugfs.h b/drivers/staging/wilc1000/debugfs.h
```

Comment:
In commit 663d1d7ee0383902ddac12a28e9010eb36dc108b I've achieved the same but in a more elegant solution, by changing the `WILC_DEBUG_REGION` value which decides which parts are being "debugged".

The default is indeed quite noisy:
```
atomic_t WILC_DEBUG_REGION = ATOMIC_INIT(INIT_DBG | GENERIC_DBG |
					 CFG80211_DBG | HOSTAPD_DBG |
					 PWRDEV_DBG);
```

I've disabled them all and the net working result is same as per your patch, but with just changing a mask instead of eliminating code, which is more maintanable in the long term and we can easily re-toggle debugging if we need to do so.